benches/src/db_lookup_times_utils/utils.rs

```diff
@@ -0,0 +1,141 @@
+use crate::{
+    db_lookup_times_utils::full_block_table::{
+        BenchDatabase,
+        BenchDbColumn,
+    },
+    utils::ShallowTempDir,
+};
+use anyhow::anyhow;
+use fuel_core::{
+    database::database_description::DatabaseDescription,
+    state::rocks_db::RocksDb,
+};
+use fuel_core_storage::kv_store::{
+    KeyValueInspect,
+    StorageColumn,
+};
+use fuel_core_types::{
+    blockchain::block::{
+        Block,
+        CompressedBlock,
+    },
+    fuel_tx::Transaction,
+    fuel_types::{
+        BlockHeight,
+        ChainId,
+    },
+};
+use itertools::Itertools;
+use rand::{
+    rngs::ThreadRng,
+    Rng,
+};
+use std::path::Path;
+use strum_macros::AsRefStr;
+
+pub type Result<T> = core::result::Result<T, anyhow::Error>;
+
+pub fn get_random_block_height(
+    rng: &mut ThreadRng,
+    block_count: BlockHeight,
+) -> BlockHeight {
+    BlockHeight::from(rng.gen_range(0..block_count.into()))
+}
+
+pub fn get_db_path() -> ShallowTempDir {
+    ShallowTempDir::new()
+}
```

Comment:
Nit: This function isn't helpful in my opinion. I'd just call `ShallowTempDir::new()` directly instead of calling it through this.